### ISSUE TITLE
Error "Could not get any response" when status = 201

### ISSUE THREAD

**Issue by joepaquette** [a85/POSTMan-Chrome-Extension#7757]
**App information:**
 - App Type: Postman for Windows
 - Postman Version: Version 7.13.0
 - OS: win32 10.0.18362 / x64

**Bug reproducible every time**
**Recent Encounter** - First time trying this task

**Describe the bug**
I have a REST API that performs a POST to add data to a database. It was recently developed returning an HTTP status code of 200 for a successful POST. The POST works consistently with data successfully added to the database. Changing the code to return 201 instead of 200 consistently results in:

> Could not get any response
> There was an error connecting to https://scrubbedbaseurl/v1/432/1/21/fhir/dstu2/DocumentReference.
>
> **NOTE:**
> Despite the reported error above, the API does actually execute correctly saving the expected data in the database successfully!
> Server debug logs show correct response, content-type, and status code (201)

I literally only changed the code to return 201 instead of 200 (in the code snippet below, I changed `$self->{Status} = 200;` to `$self->{Status} = 201;`). Resetting the status code back to 200 results in the expected results.

I'm executing the same saved request in Postman with no changes. The only change is in the server code to change the status to either 200 or 201.

**Saved Examples**
I've saved the responses as examples for both status codes within Postman. The response for 200 had the expected response body, response headers, and Status = 200. The response for 201 had an **empty** response body, the same response header values as for 200, and Status = 201.

**Expected behavior**
Postman should return the expected response body for both statuses (200 and 201) and not report an error.

**Additional context**
**Code**
This is the relevant server code block:

`

	$self->{Status} = 200;

	my $resultObj = { DOCUMENTREFERENCEID => $result->{RESULT}};

	my $jsonresult = Util::JSON::EncodeJSON({OBJECT => $resultObj});

	return $jsonresult;
`

**Screenshots**
![Response200-Body](https://user-images.githubusercontent.com/47391024/70608918-f3758680-1bce-11ea-82c1-ae05be149651.PNG)
![Response200-Headers](https://user-images.githubusercontent.com/47391024/70608923-f6707700-1bce-11ea-84c7-a6191b71c6d4.PNG)

**The only change is `200` changed to `201` in the first code line above.**

![PostmanRequest-Error](https://user-images.githubusercontent.com/47391024/70608934-f96b6780-1bce-11ea-910b-0e29dd079f4a.PNG)
![Response201-EmptyBody](https://user-images.githubusercontent.com/47391024/70608945-fbcdc180-1bce-11ea-97aa-0a7b13b32faa.PNG)
![Response201-Headers](https://user-images.githubusercontent.com/47391024/70608953-fec8b200-1bce-11ea-8ee0-01d6ae473362.PNG)

**Comment by joepaquette** [comment]
Additional note: Our team tried to reproduce the issue with a simple express app but was not able to create a reproducible example.

**Comment by joepaquette** [comment]
We've identified the cause of this issue. Postman automatically includes a set of request headers that includes `Accept-Encoding` set to gzip and deflate. There appears to be a problem with our underlying Apache server not correctly handling this. If we manually set `Accept-Encoding` to any of the following (*, identity, deflate), Postman completes successfully. If we manually set `Accept-Encoding` to gzip, we get the reported Postman error. Therefore, the issue is not Postman.
